tests/lib/spsc_pbuf/boards/nucleo_l152re.conf

```diff
@@ -0,0 +1 @@
+CONFIG_STM32_PREFETCH_DISABLE=y
```

Comment:
Instead of doing ad hoc on specific test, can we rather check if this has to be done on specific feature activation (such as `CONFIG_SPSC_PBUF`).
Maybe it's related also to a specific cache handling configuration option:
```
choice SPSC_PBUF_CACHE_HANDLING
	prompt "Cache handling"
	default SPSC_PBUF_CACHE_ALWAYS if SPSC_PBUF_USE_CACHE && !SPSC_PBUF_NO_CACHE
	default SPSC_PBUF_CACHE_NEVER if !SPSC_PBUF_USE_CACHE && SPSC_PBUF_NO_CACHE
	default SPSC_PBUF_CACHE_FLAG
```
This would have to be configured as part of the soc configuration.

This would allow to be a bit future proof on test evolution and get the feature to work for features users and not only fix a test in a specific condition.